src/components/Rezume/Rezume.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 const Rezume = ({ Resume }) => {
-  const { Name, Location, CurrentPosition, Summary } = Resume.About;
+  const { Name, Location, CurrentPosition, Summary,ProfilePicture } = Resume.About;
```

Comment:
Not done.

Reply:
@sebinfrancis7 I really want this to be merged today. This is such a nice one. I am giving you the max points for this! <3